datasets/qa/mapping.yml

```diff
@@ -4,6 +4,10 @@ Insert Account:
   - Name
   - NumberOfLocations__c
   - SLAExpirationDate__c
+  - Description
```

Comment:
@mkolodner Would you remove those from the mapping and datasets if present?

Reply:
@Szandor72  Removed and new commit published. Not sure how to put that commit into this PR....?

Reply:
I moved the commit for you. 

In order to work on this PR, you need to check out the new branch `feature/QE-Work-Michael-Kolodner`